.github/workflows/person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+
+  attr_reader :id, :name, :age
+  attr_writer :name, :age
+
```

Comment:
- [ ] I can see you are trying to set the getter and setter for the `name` and `age` instance variables. Kindly use the `attr_accessor` as it accounts for both.

  ```
  attr_accessor :name, :age
  attr_reader :id
  ```